src/samples/flow/c-api/lowlevel.c

```diff
@@ -142,7 +136,7 @@ startup(void)
     nodes[1 /* logic */].type = SOL_FLOW_NODE_TYPE_CUSTOM_NODE_TYPES_LOGIC;
     nodes[2 /* writer */].type = SOL_FLOW_NODE_TYPE_CUSTOM_NODE_TYPES_WRITER;
 #ifdef SOL_FLOW_NODE_TYPE_CONSOLE_DEFINED
```

Comment:
in that case `#include "sol-flow/console.h"`  will fail since it doesn't exist.

Internally, we should have a way to check it. External apps should do something like `AC_HEADER()` to check for it.